Formula/osmcoastline.rb

```diff
@@ -17,6 +18,7 @@ class Osmcoastline < Formula
   depends_on "gdal"
   depends_on "geos"
   depends_on "libspatialite"
+  depends_on "lz4"
```

Comment:
Why? Is this not just opportunistic linkage?

Reply:
yeah, was seeing this in the test failure.

>   ==> /opt/homebrew/Cellar/osmcoastline/2.3.1_1/bin/osmcoastline -v -o output.db input.opl
>   dyld: Library not loaded: /opt/homebrew/opt/lz4/lib/liblz4.1.dylib
>     Referenced from: /opt/homebrew/Cellar/osmcoastline/2.3.1_1/bin/osmcoastline
>     Reason: image not found

Reply:
Let's see if we can remove this in #92172 (my local 12-arm test out fine)